### PR TITLE
Add Edit feature

### LINKED ISSUE #13: Add Edit feature
User can edit from home page
User can edit from archive page

### PR DESCRIPTION
**Pull request by Raghav888**
1. User can edit from home page
2. User can edit from archive page

### CODE REVIEW

**Review by Manju-sree — APPROVED** [review]
code looks clean and formatted well, please add favicon and change title. suggesstions: In UI you can give little space between header and body and change color of header or provide box-shadow property, it will look good.

File: src/utils/noteListReducer.js [hunk 1]
```diff
@@ -142,7 +142,53 @@ export const noteListReducer = (state, action) => {
             .filter((item) => item.id === action.payload.value),
         ],
       };
+    case "EDIT_NOTE":
+      (async () => {
```
Comment (munjalSaksham71) [inline src/utils/noteListReducer.js:146]: Cool!

File: src/utils/noteListReducer.js [hunk 2]
```diff
@@ -142,7 +142,53 @@ export const noteListReducer = (state, action) => {
...
+            { notes: { ...action.payload.value } },
+            {
+              headers: {
+                Accept: "application/json",
+                "Content-Type": "application/json",
+              },
+            }
+          );
+        } catch (err) {
+          console.log(err);
```
Comment (munjalSaksham71) [inline src/utils/noteListReducer.js:159]: Maybe you can alert or create a state to show the error to user.

File: src/utils/noteListReducer.js [hunk 3]
```diff
@@ -142,7 +142,53 @@ export const noteListReducer = (state, action) => {
...
+            { archive: { ...action.payload.value } },
+            {
+              headers: {
+                Accept: "application/json",
+                "Content-Type": "application/json",
+              },
+            }
+          );
+        } catch (err) {
+          console.log(err);
```
Comment (munjalSaksham71) [inline src/utils/noteListReducer.js:183]: Same here

**Review by munjalSaksham71 — APPROVED** [review]
Overall code looks awesome and understandable.

**Pull request merged**